### PR TITLE
Some needed updates to remove broken links and out of date info

### PR DESCRIPTION
**Pull request by prushforth**
- Remove cubewerx examples
- Remove link to mapml-client library
- Add links to explainer and meta-explainer blog post
- Replace link to MapML Servlet with links to GeoServer MapML Community Module

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -197,26 +210,21 @@ <h2>Software projects</h2>
...
+        <li><a href="https://github.com/Maps4HTML/Web-Map-Custom-Element">Web-map custom element</a>
           <p>
-            The code that allows Leaflet to support MapML layers.
+            A polyfill of the proposed HTML map / layer element specification,
+            with MapML support, as a set of HTML custom elements,  using Leaflet
+            as the map rendering engine.
           </p>
         </li>
-        <li><a href="https://github.com/Maps4HTML/MapMLServer">MapML Server</a>
+        <li><a href="https://github.com/Maps4HTML/geoserver">GeoServer MapML MapML Community Module</a>
```
Comment (Malvoz) [inline index.html]: Should this say "GeoServer MapML MapML Community Module"?

Comment (prushforth) [reply index.html]: Lol no, thanks, I'll fix it Thanks for your review.

**Comment by Malvoz** [comment]
Would [RFC 5870](https://tools.ietf.org/html/rfc5870) be a good addition to the "Other links" section? (we've already got an issue in the Use Cases & Requirements repo to discuss use cases for it)

**Comment by prushforth** [comment]
> Would [RFC 5870](https://tools.ietf.org/html/rfc5870) be a good addition to the "Other links" section? (we've already got an issue in the Use Cases & Requirements repo to discuss use cases for it)

Sure thing, will add a commit in the morning.

File: index.html [hunk 2]
```diff
@@ -235,14 +255,25 @@ <h2>Working Demos</h2>
...
-        <li><a href="https://geogratis.gc.ca/mapml/static/cubewerx_cgdi_proxy_geomet_directory.html">MapML versions of GeoMet WMS</a>
+        <li><a href="https://geogratis.gc.ca/mapml/static/us_pop_density.html">Vector example</a>
+          <p>
+            This is an HTML document containing map markup inline, representing
+            vector data according to the MapML &lt;feature&gt; proposal. </a>.
+          </p>
+        </li>
+
+        
+<!--        <li><a href="https://geogratis.gc.ca/mapml/static/cubewerx_cgdi_proxy_geomet_directory.html">MapML versions of GeoMet WMS</a>
```
Comment (AmeliaBR) [inline index.html]: Instead of commenting this out, why not directly remove it?

File: index.html [hunk 3]
```diff
@@ -235,14 +255,25 @@ <h2>Working Demos</h2>
...
           <p>
             Direct access to MapML files,
             generated from live CubeWerx Environment and Climate Change data.
             These files can be dragged and dropped onto the <a href="https://geogratis.gc.ca/mapml/">MapML viewers</a>.
           </p>
         </li>
         <li><a href="https://geogratis.gc.ca/mapml/static/cubewerx_cgdi_proxy_wmts.html">Collection of MapML viewers of CubeWerx maps</a>
+        </li>-->
+        <li><a href="https://www.youtube.com/channel/UCKIv6IGwzxPF2HNhvq-VhKA" target="_blank">Maps for HTML is on YouTube</a>
+          <p>The Maps for HTML Community Group has a <a href="https://www.youtube.com/channel/UCKIv6IGwzxPF2HNhvq-VhKA" target="_blank">channel</a> on YouTube.</p>
```
Comment (AmeliaBR) [inline index.html]: This doesn't really belong in the “Working Demos” section.

Maybe add to the “About the Community Group” section? We should also add the Twitter account there, too.

Also, `target="_blank"` is unnecessary. This is a static page, the browser back button works fine if someone needs to get back to the list!

File: index.html [hunk 4]
```diff
@@ -303,6 +334,27 @@ <h2>Other links</h2>
         </li>
       </ul>
     </section>
+    <section>
+      <h2>RFCs and other relevant documents</h2>
```
Comment (AmeliaBR) [inline index.html]: Maybe make this “Related standards and document formats” and also include the OGC link? (Or call out specific OGC standards even)  And probably move it up before the general “Other Links” section?

File: index.html [hunk 5]
```diff
@@ -257,16 +288,18 @@ <h2>Other links</h2>
         <li><a href="https://www.opengeospatial.org/standards/">OGC® Standards and Supporting Documents</a>
           <p>
             The Open Geospatial Consortium (OGC) publishes standards for map-related data,
-            many of which are used or adapted for MapML.
+            some of which are used or adapted by MapML.
           </p>
         </li>
-        <li><a href="https://docs.opengeospatial.org/per/17-019.html">MapML Engineering Report (2017)</a> from the <abbr title="Open Geospatial Consortium">OGC</abbr> Testbed 13
+        <li>MapML Engineering Reports from the <abbr title="Open Geospatial Consortium">OGC</abbr> Innovation Program
```
Comment (AmeliaBR) [inline index.html]: Don't need the `target="_blank"` for the links.

Also: Maybe move these all up to the end of the “Specifications and other reports” section, so they're not lost in “Other Links”?

**Review by AmeliaBR — COMMENTED** [review]
Lots of great additions here. Makes me reconsider the overall organization though: see comments.

File: index.html (hunk 2 above)
Comment (prushforth) [reply index.html]: I was hoping that Cubewerx would consider my request to restore their services. But yes, I will delete it. We can always recover from the commit history.

File: index.html (hunk 3 above)
Comment (prushforth) [reply index.html]: OK

File: index.html (hunk 5 above)
Comment (prushforth) [reply index.html]: OK

File: index.html (hunk 4 above)
Comment (prushforth) [reply index.html]: I put the OGC standards in the renamed section as well.

**Pull request merged**